releasetasks/templates/firefox/enUS.yml.tmpl

```diff
@@ -33,7 +33,8 @@
                 - >
                   wget -O mozharness.tar.bz2 https://hg.mozilla.org/{{ repo_path }}/archive/{{ mozharness_changeset }}.tar.bz2/testing/mozharness &&
                   mkdir mozharness && tar xvfj mozharness.tar.bz2 -C mozharness --strip-components 3 && cd mozharness &&
-                  python scripts/release/beet_mover.py --no-refresh-antivirus --template configs/beetmover/en_us.yml.tmpl --platform {{ buildbot2ftp(platform) }} --version {{ version }} --app-version {{ appVersion }} --locale en-US --taskid {{ platform_info['task_id'] }} --build-num build{{ buildNumber }} --bucket {{ beetmover_candidates_bucket }}
+                  python scripts/release/beet_mover.py --no-refresh-antivirus --template configs/beetmover/en_us_signed.yml.tmpl --platform {{ buildbot2ftp(platform) }} --version {{ version }} --app-version {{ appVersion }} --locale en-US --taskid {{ platform_info['signed_task_id'] }} --build-num build{{ buildNumber }} --bucket {{ beetmover_candidates_bucket }} &&
+                  python scripts/release/beet_mover.py --no-refresh-antivirus --template configs/beetmover/en_us_unsigned.yml.tmpl --platform {{ buildbot2ftp(platform) }} --version {{ version }} --app-version {{ appVersion }} --locale en-US --taskid {{ platform_info['unsigned_task_id'] }} --build-num build{{ buildNumber }} --bucket {{ beetmover_candidates_bucket }}
```

Comment:
I'm assuming it's safe to run this script twice back to back without cleanup in between.